dask/array/tests/test_masked.py

```diff
@@ -28,6 +29,23 @@ def test_from_array_masked_array():
     assert_eq(dm, m)
 
 
+def test_copy_deepcopy():
```

Comment:
This is fine, but I also might remove this test from `ma`, and expand the `test_copy_mutate` test in `dask_array_core.py` to test an array of another dtype. That feels more like the correct place, as its testing the `copy` method, which isn't `ma` specific.

Reply:
@TAdeJong are you ok to make this change?

Reply:
@mrocklin: holiday without proper internet does not do wonders to my productivity. I will try to have a look this week as I think this is now straightforward, but my schedule is looking somewhat busy.

Reply:
Enjoy vacation!

Reply:
From @jcrist 's tone on this it seems like he's ok either way.  I think it's also ok to keep this in this testing file mostly as a regression test.  It *is* testing ma-specific functionality just to ensure that they stay masked arrays.